.storybook/theme.js

```diff
@@ -6,7 +6,7 @@ import {
   PharosColorMarbleGray50,
   PharosColorMarbleGray20,
   PharosFontFamilyBody,
-} from '../packages/pharos/lib/styles/variables.js';
+} from '../packages/pharos/lib/styles/variables';
 import '../packages/pharos/lib/styles/fonts.css';
```

Comment:
tried to standardize the exports.
what i have so far is `.js` should always drop the extension and anything else should always include the extension